MujWeb/Views/Database/Index.cshtml

```diff
@@ -0,0 +1,57 @@
+﻿@{
+    ViewData["Title"] = "Show from database";
+    
+}
+@model List<Uzivatel>
+
+<div class="text-center container">
+    <h1 class="display-4">Zobrazení z databáze</h1>
+
+    <p class="lead pb-4 pt-2">Zde je seznam všech záznamů z databáze.</p>
+</div>
+
+<div class ="container border-2 rounded bg-light d-flex flex-column">
+    <div class="py-2 px-0 d-flex justify-content-end">
+        <a asp-controller="Database" asp-action="CreateUzivatel" class="btn btn-success bg-green mx-0 p-2 active">
+            <i class="bi bi-plus"></i> Vytvořit nového uživatele
+        </a>
+    </div>
+    <div class="table-responsive">
+        <table class="p-3 mb-2 table table-light table-hover table-bordered align-content-left">
+            <thead>
+                <tr>
+                    <th class="w-25 text-nowrap">ID</th>
+                    <th class="w-25 text-nowrap">Jméno</th>
+                    <th class="w-25 text-nowrap">Datum registrace</th>
+                    <th></th>
+                </tr>
+            </thead>
+            <tbody>
+                @try {
+                    foreach (var item in Model)
```

Comment:
This still somehow does null exeption. Its probably because the Model is null, not sure why though.